res/raw/webxdc.js

```diff
@@ -15,6 +15,11 @@ window.webxdc = (() => {
 
     setUpdateListener: (cb) => (update_listener = cb),
 
+    allUpdates: () => {
+      return Promise.resolve(JSON.parse(InternalJSApi.getStatusUpdates(0)));
+    },
+
+    // deprecated, use `await updates = allUpdates()` instead
     getAllUpdates: () => {
```

Comment:
```suggestion
    getAllUpdates: () => {
    console.warn("getAllUpdates is deprecated, use `await updates = allUpdates()` instead");
```
to remind devs that it is in fact deprecated

Reply:
question is if we really want to give up the name ...